doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -67,12 +96,22 @@
 > * Number of interfaces and IP addresses to be supported by a pod
 > * IPv6 single and dual stack needs
 
+> Refer to software profile features [here](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#523-virtual-networking) and hardware profile features [here](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#543-network-resources).
+
 <a name="3.2.3"></a>
 ### 3.2.3 Container Storage Services
 
-> This shapter should discuss storage services provided by the reference architecture. 
+Containers, when running, will require ephemeral storage on which to run themselves (i.e. storage on which the unpacked container image is stored and executed from). This ephemeral storage lives and dies with the container and is a directory on the worker node on which the container is running.  Note, this means that the ephemeral storage is mounted locally in the worker node filesystem. The filesystem can also be physically external to the worker node (e.g. iSCSI, NFS, FC) but the container will still reference it as part of the local filesystem.
+
+Additional ephemeral storage can also be attached to a container through the use of Kubernetes Volumes - this can be storage from the worker node filesystem, or it can be external storage that is accessed through the use of a Volume Plugin. Volume Plugins allow the use of a storage protocol (e.g. iSCSI, NFS) or management API (e.g. Cinder) for the attaching and mounting of storage into a Pod. This additional ephemeral storage, that is attached to a container using a Kubernetes Volume, does not live and die with the container but instead follows the lifecycle of the Pod that the container is a part of. This means the ephemeral Volume persists across container restarts, whilst the Pod is still running.
```

Comment:
"This additional ephemeral storage, that is attached to a container using a Kubernetes Volume...”
Can we drop the word ephemeral from this sentence since it is coming from a volume, thus it's not ephemeral

Reply:
Well, it depends :-)
It is ephemeral in that it isn't a Persistent Volume and is destroyed when the Pod is destroyed. Happy to use an alternative term to distinguish from the storage used to execute the containers but it also needs to be clearly distinct from storage that exists independent of the Pods (i.e. Persistent Volumes).

Reply:
Yes, you're right. Thanks for describing.